#std_msgs/Header header
float64 gps_time                
int32 week                  
float64 lat                   
float64 lon                    
float64 height                
float64 lateral_speed           
float64 longitudinal_speed      
float64 down_speed              
float64 roll                  
float64 pitch                   
float64 heading                 
float64 lateral_accelerate      
float64 longitudinal_accelerate 
float64 down_accelerate        
float64 roll_speed            
float64 pitch_speed            
float64 heading_speed           
int32 flag                   
int32 n                       #RTK状态，源自POS状态量的低6位，参考枚举值：GPS_N_XXX

